core/src/main/scala/org/apache/spark/scheduler/ShuffleMapTask.scala

```diff
@@ -85,13 +82,15 @@ private[spark] class ShuffleMapTask(
       threadMXBean.getCurrentThreadCpuTime
     } else 0L
     val ser = SparkEnv.get.closureSerializer.newInstance()
-    val (rdd, dep) = ser.deserialize[(RDD[_], ShuffleDependency[_, _, _])](
+    val rddAndDep = ser.deserialize[(RDD[_], ShuffleDependency[_, _, _])](
```

Comment:
Hm .. did it require `scala.language.existentials`? If so, it's fine. If not, I think we can just revert this one - seems previous one looks slightly better ..